web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+}
+
+export const LabelsTable: FC<RouteComponentProps & LabelProps> = ({ value }) => {
+  const [showMore, doToggle] = useState(false);
+
+  const toggleMore = () => {
+    doToggle(!showMore);
+  };
+
+  return (
+    <>
+      <Button size="sm" color="primary" onClick={toggleMore}>
+        {showMore ? 'More' : 'Less'}
+      </Button>
+      {showMore ? (
+        <Table striped={true} className="table-head">
```

Comment:
I would suggest you to remove the className since it does nothing(or at least I coudn't find anyy css related to it) and to add ```bordered``` to the Table

Reply:
done